apps/condo/domains/news/hooks/useTableFilters.ts

```diff
@@ -0,0 +1,64 @@
+import {
+    NewsItem,
+    NewsItemWhereInput,
+} from '@app/condo/schema'
+import React, { useMemo } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+
+import { ComponentType, FiltersMeta } from '@condo/domains/common/utils/filters.utils'
+import { getFilter, getDayRangeFilter, getStringContainsFilter } from '@condo/domains/common/utils/tables.utils'
+
+const typeFilter = getFilter(['type'], 'array', 'string', 'in')
+const filterDateRange = getDayRangeFilter('createdAt')
+const bodyFilter = getStringContainsFilter('body')
+const titleFilter = getStringContainsFilter('title')
+
+export const useTableFilters = (): Array<FiltersMeta<NewsItemWhereInput, NewsItem>> => {
+    const intl = useIntl()
+    const TypeMessage = intl.formatMessage({ id: 'global.type' })
+    const StartDateMessage = intl.formatMessage({ id: 'global.filters.dateRange.start' })
+    const EndDateMessage = intl.formatMessage({ id: 'global.filters.dateRange.end' })
+    const CommonTypeMessage = intl.formatMessage({ id: 'news.type.common' })
+    const ЕmergencyCommonTypeMessage = intl.formatMessage({ id: 'news.type.emergency' })
+
+    const newsItemTypeOptions = useMemo(() => [
+        { label: CommonTypeMessage, value: 'common' },
+        { label: ЕmergencyCommonTypeMessage, value: 'emergency' },
```

Comment:
Why don't you use constants for news item types?
apps/condo/domains/news/constants/newsTypes.js